scene/gui/graph_edit.cpp

```diff
@@ -520,10 +602,24 @@ void GraphEdit::add_child_notify(Node *p_child) {
 		if (graph_node) {
 			graph_node->connect("slot_updated", callable_mp(this, &GraphEdit::_graph_node_slot_updated).bind(graph_element));
 			graph_node->connect("item_rect_changed", callable_mp(this, &GraphEdit::_graph_node_rect_changed).bind(graph_node));
+			_ensure_node_order_from(graph_node);
 		}
 
-		graph_element->connect("raise_request", callable_mp(this, &GraphEdit::_graph_element_moved_to_front).bind(graph_element));
-		graph_element->connect("resize_request", callable_mp(this, &GraphEdit::_graph_element_resized).bind(graph_element));
+		GraphFrame *graph_frame = Object::cast_to<GraphFrame>(graph_element);
+		if (graph_frame) {
+			background_nodes_separator_idx++;
+
+			call_deferred("move_child", graph_frame, 0);
+			call_deferred("_reorder_frame_nodes", graph_frame->get_name());
+			call_deferred("move_child", connections_layer, background_nodes_separator_idx);
```

Comment:
```suggestion
			callable_mp((Node *)this, &Node::move_child).call_deferred(graph_frame, 0);
			callable_mp((Node *)this, &Node::move_child).call_deferred(connections_layer, background_nodes_separator_idx);
```
`_reorder_frame_nodes` does not exist.